client/src/components/InputNewProduct.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { saveNewProduct } from "../redux/axiosProducts";
+import { useDispatch } from "react-redux";
+
+
+export const InputNewProduct = () => {
+    const [product, setProduct] = React.useState({ "name": "", "color": "" });
+    const dispatch = useDispatch();
+
+    const updateProduct = (event) => {
+        const value = event.target.value;
+        setProduct({
+            ...product,
+            [event.target.name]: value,
+        });
+
+    };
+
+    const handleSaveNewProduct = () => {
+        console.log(product)
+        dispatch(saveNewProduct(product))
```

Comment:
As an exercise make this component dumb, so that it's not aware of redux. In order to accomplish that InputNewProduct component must get onSubmit method, which you call here